### PR TITLE
Sara Express API

### PR DESCRIPTION
**Pull request by Sara-Borgstrom**

### CODE REVIEW

**Comment by JennieDalgren** [comment]
Add a link to the deployed version please

**Comment by Sara-Borgstrom** [comment]
https://dashboard.heroku.com/apps/book-api-technigo

**Comment by JennieDalgren** [comment]
This link ☝️is a link to your heroku dashboard. I can't access that. Update it with the actual link to your project 🙃

**Comment by Sara-Borgstrom** [comment]
Sorry, is this the one?
https://book-api-technigo.herokuapp.com/

File: server.js [hunk 1]
```diff
@@ -27,7 +31,50 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// The entire json
+app.get('/book', (req, res) => {
```
Comment (JennieDalgren) [inline server.js]: this endpoint should be called books instead of book. :)

File: server.js [hunk 2]
```diff
@@ -27,7 +31,50 @@ app.get('/', (req, res) => {
...
+// Get one book from ID
+app.get('/books/:id', (req, res) => {
+  const bookId = req.params.id
+  const book = booksData.find((item) => item.bookID === +bookId)
+  if (book) {
+    res.json(book);
+  } else {
+    res.status(404).send("No book found")
+  }
+})
```
Comment (JennieDalgren) [inline server.js]: ⭐️

File: server.js [hunk 3]
```diff
@@ -27,7 +31,50 @@ app.get('/', (req, res) => {
...
+
+// Get language filterd on language_code
+app.get('/language/:language', (req, res) => {
+  const language = req.params.language
```
Comment (JennieDalgren) [inline server.js]: you could also destructure language from req.params.
`const { language } = req.params`

File: server.js [hunk 4]
```diff
@@ -27,7 +31,50 @@ app.get('/', (req, res) => {
...
+    filteredDetails = filteredDetails.filter(item => {
+      const itemTitle = item.title.toString();
+      const itemsAuthor = item.authors.toString();
+
+      return itemTitle.includes(searchString) ||
+        itemsAuthor.includes(searchString)
+    })
+  }
+  res.json(filteredDetails)
+})
```
Comment (JennieDalgren) [inline server.js]: nice

**Pull request closed without merging**